libvast/vast/system/disk_monitor.hpp

```diff
@@ -55,6 +59,7 @@ struct disk_monitor_state {
 struct disk_monitor_config {
   size_t high_water_mark;
   size_t low_water_mark;
+  size_t step_size;
```

Comment:
These should have default values, otherwise they may contain garbage if you forget to initialize them.